igvc_navigation/launch/mapper.launch

```diff
@@ -4,8 +4,8 @@
     <!-- Map Node -->
     <node name="mapper" pkg="igvc_navigation" type="mapper" output="screen">
             <param name="topics" type="string" value="/scan/pointcloud /semantic_segmentation_cloud"/>
-            <param name="occupancy_grid_length" type="int" value="50" />
-            <param name="occupancy_grid_width" type="int" value="50" />
+            <param name="occupancy_grid_length" type="int" value="200" />
```

Comment:
this is a YUGE matrix. Why?

If our course in sim is this big then make an issue for that.

Reply:
The robot traveled too far to stay within the initial occupancy grid length. I increased the size so that I could see how it performs on the rest of the course. 200m is indeed pretty big, i'll reduce the value of this parameter on my next commit. 

What do you mean by comments inside the method? Like method explanations? i.e.

```c++

int foo(int a) 
{
    /**
    does bar on foo 
    */
    return 0;
}
```

Reply:
yes

Reply:
O.k. I think your approach is the correct one. I'll fix this.